main/kamailio/APKBUILD

```diff
@@ -229,20 +229,17 @@ for _i in db postgres sqlite dbtext mysql \
 	ldap utils tls presence lua ims outbound debugger \
 	extras json websocket authephemeral \
 	uuid ev memcached redis geoip2 jansson \
-	jsdt http_async; do
+	jsdt http_async kazoo; do
 
    subpackages="$subpackages $pkgname-$_i"
    eval "_modules=\"\$_modules \$_mod_list_$_i\""
 done
 
 source="http://www.kamailio.org/pub/kamailio/$pkgver/src/${pkgname}-${pkgver}${_suffix}.tar.gz
-	0001-musl-fixes.patch
-	0002-fix-libressl.patch
-	0003-kamdbctl.base.patch
-	0004-remove-spurious-execinfo.patch
+	0001-kamdbctl.base.patch
+	0002-remove-spurious-execinfo.patch
```

Comment:
I have not usually renamed patches in situation like this, but I guess it's ok to do so if wanted.